README.rst

```diff
@@ -17,16 +18,89 @@ high-throughput imaging experiment. The measurements are stored across thousands
 cytominer-database helps you organize these data into a single database backend, such as SQLite.
 
 Why cytominer-database?
------------------------
+=======================
 While tools like CellProfiler can store measurements directly in databases, it is usually infeasible to create a
 centralized database in which to store these measurements. A more scalable approach is to create a set of CSVs per
 “batch” of images, and then later merge these CSVs.
 
-cytominer-database ingest reads these CSVs, checks for errors, then ingests them into a database backend, including
+cytominer-database ingest reads these CSVs, checks for errors, then ingests
+them into a database backend, including
 SQLite, MySQL, PostgresSQL, and several other backends supported by odo.
 
 .. code-block:: sh
 
 	cytominer-database ingest source_directory sqlite:///backend.sqlite -c ingest_config.ini
 
 will ingest the CSV files nested under source_directory into a SQLite backend
+
+How to use the configuration file
+=================================
+The configuration file ingest_config.ini must be located in the source_directory
+and can be modified to specify the ingestion.
+There are three different sections.
+
+The [filenames] section
+-----------------------
+
+.. code-block::
+
+  [filenames]
+  image   = image.csv      #or: Image.csv
+  object  = object.csv     #or: Object.csv
+
+Cytominer-Database is currently limited to the following measurement file kinds:
+Cells.csv, Cytoplasm.csv, Nuclei.csv, Image.csv, Object.csv.
+The [filenames] section in the configuration file saves the correct basename of existing measurement files
+(this may be important in the case of inconsistent capitalization).
+# TODO: Are there any other reasons for the [filenames] section?
+
+The [database_engine] section
+-----------------------------
+
+.. code-block::
+
+  [ingestion_engine]
+  engine = Parquet      #or: SQLite
+
+The [database_engine] section specifies the backend. Possible key-value pairs are:
+**engine** = *SQLite* or **engine** = *Parquet*.
+[Potential ToDo: Delete Section and introduce a command flag]
+
+The [schema] section
+--------------------
+
+.. code-block::
+
+ [schema]
+ reference_option = sample         #or: path/to/reference/folder relative to source_directory
+ ref_fraction     = 1              #or: any decimal value in [0, 1]
+ type_conversion  = int2float      #or: all2string
+
+The [schema] section specifies how to manage incompatibilities in the table schema
+of the files.
+In that case, a Parquet file is fixed to a schema with which it was first opened,
+i.e. by the first file which is written (the reference file). To append the data
+of all .csv files of that file-kind, it is important to assure the reference file
+satisfies certain incompatibility requirements, e.g. does not miss any columns
+and all existing files can be automatically converted to the reference schema.
+This section is used only if the files are ingested to Parquet format and was
```

Comment:
This only applies for the non-default case where the user decides to build a folder with reference files, instead of determining the reference files automatically. I highlighted that case with a warning-box (further below).

Reply:
Unfortunately, github does not support .rst warning boxes or note boxes (see also: [markup issue](https://github.com/github/markup/issues/68). For now I chose the strong emphasis from the [GitHub Markdown Specs](https://github.github.com/gfm/#emphasis-and-strong-emphasis). LMK if there's something more appealing you have in mind.

Reply:
LOOKS GREAT!!!